server/backend/db_backend.go

```diff
@@ -824,3 +875,11 @@ func (self *MongoBackend) cleanUp() {
 		self.Lgr.Info("Cleaned collection", zap.String("collection", collectionName))
 	}
 }
+func (self *MongoBackend) useTransactionsByAddress() bool {
+	v, err := self.getDatabaseVersion()
+	if err != nil {
+		self.Lgr.Error("Cannot get database version", zap.Error(err))
+		return false
+	}
+	return v > 0
```

Comment:
It might be good to tie this to the migration ID directly, e.g. `return v >= migrationIDTransactionsByAddress`, or with more refactoring `return v >= migrationTransactionsByAddress.ID`.